src/screens/OrganizationPeople/OrganizationPeople.tsx

```diff
@@ -1,30 +1,51 @@
 import { useLazyQuery } from '@apollo/client';
 import dayjs from 'dayjs';
 import React, { useEffect, useState } from 'react';
-import { useLocation } from 'react-router-dom';
-import { Container, Form } from 'react-bootstrap';
+import { Link, useLocation } from 'react-router-dom';
+import { Button, Dropdown, Form } from 'react-bootstrap';
 import Col from 'react-bootstrap/Col';
 import Row from 'react-bootstrap/Row';
-
 import {
   ORGANIZATIONS_MEMBER_CONNECTION_LIST,
   USER_LIST,
 } from 'GraphQl/Queries/Queries';
 import NotFound from 'components/NotFound/NotFound';
-import OrgAdminListCard from 'components/OrgAdminListCard/OrgAdminListCard';
-import OrgPeopleListCard from 'components/OrgPeopleListCard/OrgPeopleListCard';
 import OrganizationScreen from 'components/OrganizationScreen/OrganizationScreen';
 import PaginationList from 'components/PaginationList/PaginationList';
-import UserListCard from 'components/UserListCard/UserListCard';
 import { useTranslation } from 'react-i18next';
 import styles from './OrganizationPeople.module.css';
-
 import { toast } from 'react-toastify';
+import { Search, Sort } from '@mui/icons-material';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell, { tableCellClasses } from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+import { styled } from '@mui/material/styles';
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
Remove this comment

Reply:
hey @rishav-jha-mech when I am removing I am getting this error ,I am using the functions as wrapper so it needs to be like this,  how should i fix it 
![image](https://github.com/PalisadoesFoundation/talawa-admin/assets/90516956/f154df35-69f7-495a-ae7c-70ffa053ad0c)

Reply:
@duplixx so have you tried using camelCase or UPPER_CASE variable name ?

Reply:
@rishav-jha-mech yeah i did that